helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -36,14 +36,7 @@ rules:
   verbs:
     - get
     - list
-    - watch
-- apiGroups:
-    - application.giantswarm.io
-  resources:
-    - apps
-  verbs:
-    - get
-    - list
+    - patch
```

Comment:
Diff is a bit strange. This adds patch support for appcatalog CRs. 

There were duplicate rules for the apps CRs which get removed.